ait/core/notify.py

```diff
@@ -47,24 +47,25 @@ def send_text_alert(msg, recipients=None):
 def _send_email(message, recipients):
     ''''''
     if type(recipients) != list:
-        recipients = [recipients]
+    recipients = [recipients]
 
     if len(recipients) == 0 or any([i is None for i in recipients]):
         m = (
             'Email recipient list error. Unable to send email. '
             'Recipient list length: {} Recipients: {}'
-        ).format(len(recipients), ', '.join(recipients))
-        log.error(m)
-        return
+    ).format(len(recipients), ', '.join(recipients))
```

Comment:
This indent change looks wrong to me. Should be the way it was previously to keep the message logging / return a part of the if check. As it is I don't think this would function as expected?